src/webrtc/call.ts

```diff
@@ -2465,6 +2498,8 @@ export class MatrixCall extends TypedEventEmitter<CallEvent, CallEventHandlerMap
     private async terminate(hangupParty: CallParty, hangupReason: CallErrorCode, shouldEmit: boolean): Promise<void> {
         if (this.callHasEnded()) return;
 
+        if (this.stopVideoTrackTimer !== undefined) clearTimeout(this.stopVideoTrackTimer);
```

Comment:
Could this please be at the same place as the other `clearTimeout()`s

Reply:
Oops, yes